pkg/intltool

```diff
@@ -7,11 +7,12 @@ perl
 perl5-XML-Parser
 
 [vars]
-filesize=140340
-sha512=87769800d88fe16965e59b919d1c219e928fcc4e6708f07ecba7f97c33aa5e4955635d487ef1632c94632730a9755de4776c1b2d553ae0860a038ad250153955
+filesize=162286
+sha512=4c93cb26592ed0b1622d3b7800b5d6622ffa36f58ac73f2ef0bddfab522565fdfb8fa73d81914b9a202f1d62bc995a08960f8adb9f62c86918e75498e85fbfdf
+pkgver=1
```

Comment:
a missing pkgver variable means it defaults to 1. i changed this to 2 in the version i committed in my local working tree.